hack/jenkins/release_github_page.sh

```diff
@@ -121,6 +122,7 @@ FILES_TO_UPLOAD=(
     'minikube-windows-amd64.sha256'
```

Comment:
While you are here, I'm pretty sure this should be minikube-windows-amd64.exe.sha256

Reply:
Yes, that looks right.

Reply:
Added https://github.com/kubernetes/minikube/pull/3838